Kingfisher/UIImageView+Kingfisher.swift

```diff
@@ -134,6 +159,7 @@ public extension UIImageView {
 
 // MARK: - Associated Object
 private var lastURLkey: Void?
+private var showIndicatorWhenLoading: Bool?
```

Comment:
This will be set globally to all UIImageView. Meaning that if you set it to true to 1 UIImage, It will be true to all the other instances. One more reason to have the the activity indicator as a item in the options.